packages/aws-cdk-lib/aws-appsync/lib/graphqlapi.ts

```diff
@@ -621,6 +630,9 @@ export class GraphqlApi extends GraphqlApiBase {
     if (props.resolverCountLimit !== undefined && (props.resolverCountLimit < 0 || props.resolverCountLimit > 10000)) {
       throw new Error('You must specify a resolver count limit between 0 and 10000.');
     }
+    if (props.ownerContact !== undefined && (props.ownerContact.length > 256)) {
```

Comment:
Since a String could potentially be a Token, I think it would be better to include a check to see if it is a Token. 
Although I think the likelihood of a Token exceeding 256 characters is low, it's better to be safe.

Ref: https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#throwing-exceptions

```suggestion
    if (!Token.isUnresolved(props.ownerContact) && props.ownerContact !== undefined && (props.ownerContact.length > 256)) {
```